frontend/src/cards/ui/HighestLowestList.tsx

```diff
@@ -98,6 +101,18 @@ export function HighestLowestList(props: HighestLowestListProps) {
         </Grid>
       </div>
       <p>All rates are reported as: {props.metricConfig.fullCardTitleName}</p>
+      <p>
+        Consider the possible impact of
+        <Button
+          onClick={() =>
```

Comment:
I wonder if it's better to just do a plain document.getElementById here rather than just pass the reference down two layers?

Reply:
much simpler, done